### PR TITLE
(Fix) Don't display MoC for Sokol network if MoC is removed

### PR DESCRIPTION
**Pull request by varasev**
- (Mandatory) Description
This PR closes small fixes related to https://github.com/poanetwork/poa-dapps-validators/issues/61 and performs `npm update web3`.

- (Mandatory) What is it: (Fix), (Feature) or (Refactor)
(Fix)

### CODE REVIEW

**Review by vbaranov — COMMENTED** [review]
I believe we should merge this PR after updated smart-contracts will be implemented on Sokol and then on Core. Because the current `PoaNetworkConsensusContract` has no implementation to track removing of MoC.

**Comment by varasev** [comment]
We could merge it before because the new code checks whether `isMasterOfCeremonyRemoved` method exists in `PoaNetworkConsensus`: https://github.com/varasev/poa-dapps-validators/blob/38663f657cb26995ff8aca1946d8ca7f8affd5df/src/contracts/PoaConsensus.contract.js#L21

It has been done to support the current version of `PoaNetworkConsensus`. This code has been tested with the current `PoaNetworkConsensus` in `Sokol` and `Core` networks.

**Comment by vbaranov** [comment]
@varasev then let's create for Sokol first

**Comment by varasev** [comment]
@vbaranov i.e., you suggest adding some code to check whether we are in `Sokol` in the place where  `isMasterOfCeremonyRemoved` is called, right?

**Comment by vbaranov** [comment]
yes, if it is not a big change, let's apply changes in this PR only when we are in Sokol. Just to be sure, those changes don't break the functionality and then we will extend it to Core as well.

**Comment by varasev** [comment]
@vbaranov ok.

**Comment by varasev** [comment]
I've added checking for Sokol network in the code: https://github.com/poanetwork/poa-dapps-validators/pull/62/commits/81a19e2f331c33b32223422c47c5f858c85a1a64

File: src/contracts/Metadata.contract.js [hunk 1]
```diff
@@ -82,12 +82,12 @@ export default class Metadata {
     let createdDate = validatorData.createdDate > 0 ? moment.unix(validatorData.createdDate).format('YYYY-MM-DD') : ''
     let updatedDate = validatorData.updatedDate > 0 ? moment.unix(validatorData.updatedDate).format('YYYY-MM-DD') : ''
     let expirationDate = validatorData.expirationDate > 0 ? moment.unix(validatorData.expirationDate).format('YYYY-MM-DD') : ''
-    if (validatorData.zipcode.length === 4) {
-      validatorData.zipcode = "0" + validatorData.zipcode;
-    }
-    if (validatorData.zipcode === "0") {
-      validatorData.zipcode = '';
-    }
+    //if (validatorData.zipcode.length === 4) {
```
Comment (vbaranov) [inline src/contracts/Metadata.contract.js]: @varasev could you remind what is the intention to comment these lines in this PR?

Comment (varasev) [reply src/contracts/Metadata.contract.js]: @vbaranov These lines worked for the old ValidatorMetadata when `zipcode` had `uint256` type, and we had to add a leading zero for 4-digit zip codes for that reason. Now `zipcode` has `bytes32` type and contains all the necessary digits for zip code including a leading zero, so now these lines don't have any sense.

Comment (vbaranov) [reply src/contracts/Metadata.contract.js]: Maybe, let's remove these lines then?

Comment (varasev) [reply src/contracts/Metadata.contract.js]: Yes, I've removed them: https://github.com/poanetwork/poa-dapps-validators/pull/62/commits/8aa5a6edb25d492d89422062219040db264d3ba1

**Review by vbaranov — APPROVED** [review]

**Pull request merged**